src/screens/ReportCar.js

```diff
@@ -1,19 +1,80 @@
 import React, { useState } from "react";
 import get from "lodash/get";
 import { View, Text, StyleSheet, TouchableOpacity, Image } from "react-native";
-import {Layout} from "../components/Layout";
-import {Button} from "../components/Button";
+import { Layout } from "../components/Layout";
+import { Button } from "../components/Button";
+import * as API from "../utils/API";
+import { TextInput } from "react-native-gesture-handler";
 
 const ReportCar = ({ navigation, props, route }) => {
   const imageURI = get(route, "params.imageURI");
+  const [plateNumber, setplateNumber] = useState("none");
+  const [mediaUrl, setMediaUrl] = useState();
+  const [lat, setLat] = useState("40");
+  const [long, setLong] = useState("40");
+
+  var photo = {
+    uri: imageURI,
+    type: "image/jpeg",
+    name: "photo.jpg",
+  };
+
+  const setCarImg = async (img) => {
+    try {
+      const fetchedReport = await API.setCarImage(img);
+      if (fetchedReport) {
+        setplateNumber(fetchedReport.platesProposal);
+        setMediaUrl(fetchedReport.url);
+      }
+    } catch (err) {
+      console.error("Err: ", err);
+    }
+  };
+
+  const submitReport = async (p1, p2, p3, p4) => {
```

Comment:
please use better naming here for func args, it should be `lat, long, mediaUrl, plateNumber`